pkg/crc/validation/validation.go

```diff
@@ -61,6 +61,7 @@ func ValidateBundlePath(bundlePath string, preset crcpreset.Preset) error {
 	}
 
 	userProvidedBundle := filepath.Base(bundlePath)
+	userProvidedBundle = bundle.GetBundleNameWithExtension(userProvidedBundle)
```

Comment:
I have a hard time understanding why this change is needed from the commit log :-/

Reply:
@cfergeau it is there to make use of `GetBundleNameWithExtension` function, Removing it from this commit.